tests/valgrind-check/valgrind.sh

```diff
@@ -0,0 +1,193 @@
+#!/bin/bash
+
+function print_ps {
+    set +e
+    echo "CFEngine processes:"
+    ps aux | grep [c]f-
+
+    echo "Valgrind processes:"
+    ps aux | grep [v]algrind
+    set -e
+}
+
+function stop_daemons {
+    echo "Stopping cfengine daemons"
+    pkill -f cf-serverd  || echo "Did not kill cf-serverd"
+    pkill -f cf-execd    || echo "Did not kill cf-execd"
+    pkill -f cf-monitord || echo "Did not kill cf-monitord"
+}
+
+function no_errors {
+    set +e
+    grep -i "error" $1
+    grep -i "error" $1 && exit 1
```

Comment:
The second `grep` should use `-q`, maybe?

Reply:
Yep, thanks.